storage/src/vespa/storage/persistence/filestorage/mergestatus.cpp

```diff
@@ -21,12 +80,14 @@ MergeStatus::~MergeStatus() = default;
 bool
 MergeStatus::removeFromDiff(
         const std::vector<api::ApplyBucketDiffCommand::Entry>& part,
-        uint16_t hasMask)
+        uint16_t hasMask,
```

Comment:
Consider adding a function comment that clarifies that this is the _per-request_ mask, not the global mask (and maybe also why that is why we need to additionally provide the original node set)